### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by GioAceto**
- Changes Twitter logos to X logos
- Fixes image block centering and sizing
- Adds autofocus to the search modal

### CODE REVIEW

File: wp-setup/dev-theme-site.db
Comment (bwaltz) [inline wp-setup/dev-theme-site.db:1]: @GioAceto What is this file?

Comment (GioAceto) [reply wp-setup/dev-theme-site.db:1]: I've removed the folder

File: vue/components/SearchModal.vue [hunk 1]
```diff
@@ -219,24 +219,21 @@
                           class="wp-social-link wp-social-link-twitter wp-block-social-link"
                         >
                           <a
-                            href="https://twitter.com/colbycollege"
-                            aria-label="Twitter: https://twitter.com/colbycollege"
+                            href="https://x.com/colbycollege"
+                            aria-label="Twitter: https://x.com/colbycollege"
```
Comment (bwaltz) [inline vue/components/SearchModal.vue:223]: @GioAceto can we change "Twitter:" to "X:" in that aria-label

File: css/blocks.css [hunk 2]
```diff
@@ -1269,7 +1269,66 @@ h6.has-background {
 }
 
 .wp-block-image.aligncenter {
+  margin: 6rem auto;
```
Comment (bwaltz) [inline css/blocks.css:1272]: @GioAceto this seems to have increased the margin on center aligned images. Can we remove this style and see what it looks like?

**Pull request merged**